### DISCUSSION TITLE
The experiment stops, the solver seems to be stuck, and the simulation does not report an error, what should I do?

### DISCUSSION THREAD

**Discussion by RobinBluehub** [pybamm-team/PyBaMM#2842] [Q&A]
At present, I am doing a fast charging experiment based on pybamm, but I found that the experiment is easy to get stuck, that is, the solver has been stuck in the discharge process and stagnated. The solver does not report an error, which makes it difficult for me to find out what the problem is. I found it useless by changing parameters such as dt_max. I wonder if you have encountered such a situation? I hope the pybamm development team can answer my confusion!
Best regards
Robin
!
[Uploading image.png…]()

**Comment by Mrzhang-hub** [comment]
I have met the same problem when I set the upper voltage limit too high approximately 5.3V. The reason I set this high is that I do not want the simulation stop when the voltage exceeds the maximum voltage 4.2 V. However, the solver will be stuck and stagnated. I donot know what happens. But I found another way to avoid overvoltage through #2687 .

**Reply by RobinBluehub** [reply]
I want to achieve a super fast charging experiment based on the SPMe model, coupling the SEI and lithium plating aging models. However, after the model runs for 30 cycles, the solver stalls if the charge rate exceeds 5.2C. I'm wondering if it's because the SPMe model doesn't support high-rate charging?

**Reply by Mrzhang-hub** [reply]
Which set of the parameters of SPMe are you using? `chen2020`?  Because the battery type of `chen2020`  is LGM21700 and the maximum C-rate recommended by the manufacturer is 1.5C. You can refer to this post #1632. My research interest is similar with yours, maybe we can discuss more about this. My email address is 202132131114@cqu.edu.cn.

**Reply by RobinBluehub** [reply]
Check mails.
